zmon_aws_agent/aws.py

```diff
@@ -783,3 +783,44 @@ def get_limits(region, acc, apps, elbs):
     entity.update(limits)
 
     return entity
+
+
+def get_sqs_queues(region, acc):
+    sqs_queues = []
+
+    try:
+        sqs_client = boto3.client('sqs', region_name=region)
+        list_queues_response = call_and_retry(lambda: sqs_client.list_queues())
+        for queue_url in list_queues_response['QueueUrls']:
+            try:
+                attributes_response = call_and_retry(
+                    lambda: sqs_client.get_queue_attributes(QueueUrl=queue_url, AttributeNames=['All']))
+                attributes = attributes_response['Attributes']
+                queue_arn = attributes['QueueArn']
+                match = re.match(r"^arn:aws:sqs:(?P<region>.+):(?P<account>.+):(?P<name>.+)$", queue_arn)
```

Comment:
Also wondering, if we're left with only capturing the name, if a simple split by ':' and taking the last element wouldn't be enough and faster